tests/monitoring/monitoring.go

```diff
@@ -160,12 +154,17 @@ var _ = Describe("[Serial][sig-monitoring]Monitoring", Serial, decorators.SigMon
 			kv, err := virtClient.KubeVirt(originalKv.Namespace).Get(context.Background(), originalKv.Name, metav1.GetOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
-			kv.Spec.CustomizeComponents = v1.CustomizeComponents{}
+			patchData := []map[string]interface{}{
```

Comment:
Can you please use functions from `patch.go`? For example:
```
patchData, err := patch.New(patch.WithRemove("/spec/customizeComponents")).GeneratePayload()
Expect(err).ToNot(HaveOccurred())
```

Reply:
Done